### DISCUSSION TITLE
✅ exclude specific background image from being lazy loaded

### DISCUSSION THREAD

**Discussion by wpshushu** [elementor/elementor#23574] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Background images used as page banners at the top can become visible as soon as possible.

### Describe the solution you'd like

Provide a toggle next to the background image picker that allows to turn off its lazy loading.

**Comment by sheaky278** [comment]
It would be better if Elementor adds a toggle button to disable lazy loading for background image instead of entering the class name. Not everyone can remember the class name and they will have to look for it to disable the feature. 

Reference: 

![Untitled-1](https://github.com/elementor/elementor/assets/77334071/9144f54a-df4c-4502-a065-6bd8806deb84)

**Comment by nicholaszein** [comment]
Hi!

This is now a possibility! 🙌 

You can use the `e-no-lazyload` class on any **Container** element to disable the lazy-loading of background images, **since version 3.22**. Moreover, background images above the fold are not lazy-loaded by default.

You can learn more about it in [this article](https://developers.elementor.com/elementor-3-21-beta/#:~:text=to%20apply%20the%20%E2%80%9C-,e%2Dno%2Dlazyload,-%E2%80%9D%20label%20in%20the).

I'm closing this discussion as solved!

Kind regards
